great_expectations/self_check/util.py

```diff
@@ -2774,7 +2775,7 @@ def evaluate_json_test_cfe(validator, expectation_type, test, raise_exception=Tr
     return (result, error_message, stack_trace)
 
 
-def check_json_test_result(test, result, data_asset=None) -> None:
+def check_json_test_result(test, result, data_asset=None) -> None:  # noqa: C901 - 49
```

Comment:
@Kilo59 What does `# noqa: C901 - 49` accomplish here?  Thanks.

Reply:
https://superconductive.atlassian.net/wiki/spaces/SUP/pages/435978345/Static+Analysis+Type-Checking#Cyclomatic-Complexity

It is suppressing mccabe complexity warnings.

Reply:
🙇